src/dfx/src/lib/identity/pem_encryption.rs

```diff
@@ -21,7 +20,6 @@ pub fn load_pem_file(path: &Path, config: Option<&IdentityConfiguration>) -> Dfx
     let content = std::fs::read(path)
         .with_context(|| format!("Failed to read {}.", path.to_string_lossy()))?;
     let content = maybe_decrypt_pem(content.as_slice(), config)?;
-    identity_utils::validate_pem_file(&content)?;
```

Comment:
What's the rationale for validating the pem file at only one call point, rather than here?

Reply:
Because in two out of the three code paths that call this, right after loading it, it is parsed into an identity, which is also the validation step. So the same work is being performed twice.

Reply:
> in two out of the three code paths
Okay.  Then the third case is export.  Is it intentional then that in that case, we no longer validate the key before exporting it?

Reply:
Ah, I seem to have misremembered, I was thinking of *import*.